start.py

```diff
@@ -1,47 +1,230 @@
-def print_board(board):
-    print("-------------")
-    for row in board:
-        for elem in row:
-            print(elem, end=' ')
-        print("\n-------------")
+import random
 
+class Player:
+    def __init__(self, name):
+        self._name = name
+        self._board = None
 
-def print_available_pieces(pieces):
-    print("Available Pieces:")
-    sorted_pieces = sorted(pieces)
-    for piece in sorted_pieces:
-        print(piece, end=" ")
+    def join_board(self, board):
+        self._board = board
```

Comment:
nie podoba mi się to dzielenie jednej tablicy pomiędzy trzema obiektami, bo np jeśli ktoś dopisze jakąś metodę i okaże się że te metoda robi coś złego z tablicą, to tym samym psuje właściwą tablicę. a to że dostęp do tej samej tablicy mają aż trzy obiekty utrudnia zlokalizowanie takiego bugu

Reply:
[Zbiorcza odpowiedź](https://github.com/wikimody/Quarto-project/pull/2#issuecomment-1843002606)